Adding cost management permissions (#7454)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/manage-access/enterprise-permissions.md

- Read access to public models. 
- No other access. 

</Expandable>
<Expandable alt_header="Cost Management admin">

> A specialized permission set intended to be used standalone or in conjunction with other permission sets to grant write access to the Cost Management dashboard. Great for granting external (non-dbt) users and analysts access to the dashboard for customizing views of warehouse spend information.

Author: ```suggestion
A specialized permission set intended to be used standalone or in conjunction with other permission sets to grant write access to the Cost Management dashboard. Used for granting access to the dashboard for customizing views of warehouse spend information. Can be paired with [Analyst-licensed users](/docs/cloud/manage-access/seats-and-users).
```

> A specialized permission set intended to be used standalone or in conjunction with other permission sets to grant write access to the Cost Management dashboard. Used for granting access to the dashboard for customizing views of warehouse spend information. Can be paired with [Analyst-licensed users](/docs/cloud/manage-access/seats-and-users).

Reviewer: ```suggestion
A specialized permission set intended to be used standalone or in conjunction with other permission sets to grant write access to the Cost Management dashboard. Used for granting access to the dashboard that enables you to customize views of warehouse spend information. Can be paired with [Analyst-licensed users](/docs/cloud/manage-access/seats-and-users).
```

Author: Re-reading all of this and I want to remove any ambiguity as this is a purpose build permission set. How does this sound:
```suggestion
A specialized permission set intended to be used standalone or in conjunction with other permission sets to grant write access to the Cost Management dashboard. We created this permission set so you can access the dashboard and customize the views without granting admin (write) access to other areas of the <Constant name="dbt_platform" />. Can be paired with [Analyst-licensed users](/docs/cloud/manage-access/seats-and-users).
```

Reviewer: Sounds great! Can you use `<Constant name="dbt_platform" />`?

Notable features:
- Write access to the full set of Cost Management dashboard features.

> - Read access to settings related to Cost Mangement dashboard configuration

Reviewer: Should this be `write` access?
```suggestion
- Write access to settings related to Cost Management dashboard configuration.
```

> - Write access to settings related to Cost Management dashboard configuration.

Author: ```suggestion
- Read access to settings related to Cost Management dashboard configuration.
```

- No other access.

</Expandable>
<Expandable alt_header="Cost Management viewer">

> A specialized permission set intended to be used standalone or in conjunction with other permission sets to grant read-only access to the Cost Management dashboard. Great for granting external (non-dbt) users access to the dashboard for viewing warehouse spend information.

Author: ```suggestion
A specialized permission set intended to be used standalone or in conjunction with other permission sets to grant read-only access to the Cost Management dashboard. Used for granting read-only access to the dashboard. Can be paired with [Analyst-licensed users](/docs/cloud/manage-access/seats-and-users).
```

Notable features:

> - Read access to the full set of Cost Management dashboard features

Reviewer: ```suggestion
- Read access to the full set of Cost Management dashboard features.
```

> - Read access to settings related to Cost Mangement dashboard configuration

Reviewer: ```suggestion
- Read access to settings related to Cost Management dashboard configuration.
```
